stacker/tests/blueprints/test_base.py

```diff
@@ -181,3 +183,48 @@ class TestBlueprint(Blueprint):
         parameters = blueprint.get_parameters()
         self.assertEqual(parameters["Param1"], 1)
         self.assertEqual(parameters["Param2"], "Test Output")
+
+    def test_resolve_string_comma_delmited_list(self):
+        provider = MagicMock()
+        context = MagicMock()
+        provider.get_output.return_value = "Test Output"
+        output = resolve_string("some-stack::Output, other-stack::Output",
```

Comment:
We should ditch this old syntax when using `blueprint_parameters` in preference for the new template syntax.